_includes/tutorials/connect-add-key-to-source/ksql/markup/dev/create-table.adoc

```diff
@@ -1,6 +1,6 @@
 Now that we have a topic with data in from the source system and the keys appropriately set, we can declare a ksqlDB table over the topic.
 
-We only need to specify the datatype of the table's key (`ROWKEY`) - the rest of the schema is picked up automagically from the Schema Registry since we're using Avro to serialize the value part of the payload. 
+We only need to specify the datatype of the table's key (`CITY_ID`)- the rest of the schema is picked up automagically from the Schema Registry since we're using Avro to serialize the value part of the payload.
```

Comment:
```suggestion
We only need to specify the datatype of the table's key (`CITY_ID`) – the rest of the schema is picked up automagically from the Schema Registry since we're using Avro to serialize the value part of the payload.
```